apps/voting/contracts/Voting.sol

```diff
@@ -213,10 +269,27 @@ contract Voting is IForwarder, AragonApp {
         return _canVote(_voteId, _voter);
     }
 
-    function getVote(uint256 _voteId)
-        public
-        view
-        voteExists(_voteId)
+    /**
+    * @dev Initialization check is implicitly provided by `voteExists()` as new votes can only be
+    *      created via `newVote(),` which requires initialization
+    */
+    function canVoteOnBehalfOf(uint256 _voteId, address _principal, address _representative) public view voteExists(_voteId) returns (bool) {
```

Comment:
As we already have the internal one, wouldn't it make sense to make this one external?

Reply:
I was just following the visibility pattern we are using for the rest of these "kind of getter" functions, thoughts?